js/larger.js

```diff
@@ -0,0 +1,10 @@
+let num1 = 20
+let num2 = 20.000001
+
+if (num1 > num2) {
+    console.log("The larger number of " + num1 + " and " + num2 + " is " + num1 + ".")
+} else if (num1 < num2) {
+    console.log("The larger number of " + num1 + " and " + num2 + " is " + num2 + ".")
+} else {
+    console.log(num1 + " and " + num2 + " are equally large numbers.")
+}
```

Comment:
looks great ! - a word of caution: JS - many compile languages actually - don't like floats / decimals and we can run into some weird bugs if we don't know about them see .1 + .2